solutions/usaco-671.mdx

```diff
@@ -84,4 +84,63 @@ int main()
 
 </CPPSection>
 
+<PySection>
+
+```py
+from collections import defaultdict, deque
+
+HORIZ, VERTI = 0, 1  # direction of laser beam (horizontal or vertical)
```

Comment:
if you're using this to indicate state, might i recommend a boolean or enum instead